packages/cloud-cognitive/src/components/Datagrid/Datagrid/DatagridEmptyBody.js

```diff
@@ -8,7 +8,10 @@
 import React from 'react';
 import { pkg } from '../../../settings';
 import { DataTable } from '@carbon/react';
-import { NoDataEmptyState } from '../../EmptyStates/NoDataEmptyState';
+import {
+  NoDataEmptyState,
+  ErrorEmptyState,
+} from '../../EmptyStates/NoDataEmptyState';
```

Comment:
This import needs to be updated

```suggestion
} from '../../EmptyStates';
```